cpp/data/installer/scripts/install_centos_packages.sh

```diff
@@ -44,9 +44,10 @@ dnf --assumeyes update
 InstallIfMissing "ca-certificates"
 dnf --assumeyes install \
     ant bc cifs-utils clang crontabs ftp gawk gcc-c++ git glibc-all-langpacks ImageMagick incron java-1.8.0-openjdk-devel \
-    jq libcurl-devel libdb-devel libsq3-devel libstemmer-devel libuuid-devel libwebp libxml2-devel libxml2 libxslt lsof lz4 make mariadb \
-    mariadb-devel mariadb-server mariadb-server-utils mod_ssl mutt openssl-devel pcre-devel policycoreutils-python-utils \
-    poppler-utils python3 python3-pexpect python3-pyflakes rpmdevtools rsync sqlite sudo tidy unzip xerces-c-devel yaz
+    jq libcurl-devel libdb-devel libsq3-devel libpq-devel libstemmer-devel libuuid-devel libwebp libxml2-devel libxml2 \
+    libxslt lsof lz4 make mariadb mariadb-devel mariadb-server mariadb-server-utils mod_ssl mutt openssl-devel pcre-devel \
+    policycoreutils-python-utils poppler-utils postgresql postgresql-server python3 python3-pexpect python3-pyflakes \
```

Comment:
Since we only want the postgresql support to connect to external servers, we do not want to install a full-fledged postgresql-server on all machines right now (including production). Or is it mandatory if we want to use the API?